src/components/common/Header/Header.js

```diff
@@ -20,17 +28,18 @@ const Header = () => {
             id: "search.products",
           })}
         />
-        <button className="search-button">
-          <img src={searchButton} alt="Logo" className="button-icon" />
+        <button className="search-button" disabled={loading}>
+          <img src={searchButton} alt="" className="button-icon" />
```

Comment:
This alt text shouldn't be removed due to the lack of other label.
Maybe change it to something descriptive like "Search for products" to ensure accessible technologies can interpret that.
```suggestion
          <img src={searchButton} alt="Search for products" className="button-icon" />
```